public/js/index.js

```diff
@@ -147,10 +147,11 @@
     });
   });
 
-  $('form.donation-variable').on('submit', function(event) {
+  $('button.donation-variable').on('click', function(event) {
     event.preventDefault();
+    var productId = $(this).data('product-id');
```

Comment:
it annoys me in jquery that both `data('productId')` and `data('product-id')` work, completely unrelated to this though